modules/rosa-list-objects.adoc

```diff
@@ -480,6 +524,42 @@ Describe an add-on named `dbaas-operator`.
 ----
 $ rosa describe addon dbaas-operator
 ----
+[id="rosa-describe-break-glass-credential_{context}"]
+== describe break glass credential
+
+Shows the details for a break glass credential for a specific cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa describe break-glass-credential --id=<break_glass_credential_id> --cluster=<cluster_name>| <cluster_id> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--id
+|Required: The ID (string) of the break glass credential.
+|--cluster
+|Required: The name or ID (string) of the cluster.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--profile
+|Specifies an AWS profile (string) from your credentials file.
+|===
```

Comment:
What's your opinion on adding here the command for retrieving kubeconfig for the break-glass-credential using the command "rosa describe break-glass-credential --id=<break_glass_credential_id> --cluster=<cluster_name> --kubeconfig" in a note ?

Reply:
Or you can add --kubeconfig flag in optional argument 
`--kubeconfig       Retrieve the kubeconfig from the break glass credential`